src/repo.rs

```diff
@@ -2721,6 +2722,46 @@ impl Repository {
             Ok(())
         }
     }
+
+    /// Perform a revert.
```

Comment:
looking at the two comments it sounds misleading as If this is not changing the index. reading up on https://libgit2.org/libgit2/#HEAD/group/revert/git_revert and https://libgit2.org/libgit2/#HEAD/group/revert/git_revert_commit it sounds more like both will alter the index. 
some of the methods in git2-rs went about plain duplicating the doc comment from the original. maybe this makes sense here aswell?

Reply:
Yes, both change the index. One changes only the index. Will duplicate the doc comment :+1: